src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -102,6 +102,8 @@ export type MultiComboboxProps<T> = Omit<NativeAttributes<'input'>, 'value' | 'o
   /** The maximum height (in pixels) of the MultiCombobox dropdown. Defaults to 300. */
   maxHeight?: number;
 
+  /** The maximum height (in pixels) of the MultiCombobox dropdown. Defaults to 300. */
+  maxContainerHeight?: number;
```

Comment:
@alexmylonas I think we need to change the comment here since it's basically the same as `maxHeight` 😄